### ISSUE TITLE
Deeper data review

### ISSUE THREAD

**Issue by zenrabbit** [zenrabbit/Comprehensive_shrub_contrasts#1]
### Data work by JB. 

[ ] redundant, non-unique plot numbers

**Comment by jennabraun** [comment]
non-unique plot numbers: I confirmed that concatenating plot_number, region, year, site, rep and canopy results into a new ID results in one observation per species per new ID
